template/config/vue-i18n/src/i18n/index.js

```diff
@@ -0,0 +1,15 @@
+import { createI18n } from 'vue-i18n'
+
+const messages = {
+  en: {
+    // English Transtlations
+  }
+}
+
+export default createI18n({
+  legacy: false,
+  globalInjection: true,
+  locale: 'en',
+  fallbackLocale: 'en',
+  messages
+})
```

Comment:
Also, when used in TS, there are additional informations that should be added to have a proper typechecking:
https://vue-i18n.intlify.dev/guide/advanced/typescript.html